### PR TITLE
Fix unmet plyr dependency

### PR DESCRIPTION
**Pull request by ajzz**
This approach adds the required dependency.

### CODE REVIEW

File: R/fb_search.R [hunk 1]
```diff
@@ -73,6 +73,7 @@ fbad_get_search <- function(
     }
 
     if (!inherits(res, 'data.frame')) {
+        library(plyr)
```
Comment (daroczig) [inline R/fb_search.R]: Instead of loading the package, we should rather import `ldply` at https://github.com/cardcorp/fbRads/blob/master/R/zzz.R#L9

Comment (ajzz) [inline R/fb_search.R]: Totally missed the directives there. Will push changes shortly.

**Comment by ajzz** [comment]
Ready

```
> fbad_get_search(fbacc = fbacc, q = 'rstats', type = "adinterest")
             id                     name audience_size path description
1 6003212345926 R (programming language)       1308280 NULL          NA
```

**Comment by daroczig** [comment]
Wow, 1.3M FB users interested in #rstats? That's impressive :sunglasses:

**Pull request merged**